test/feature/test_disk.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+from kornia.feature.disk import DISK
+
+
+class TestDisk:
+    def test_smoke_depth(self, device):
+        disk: DISK = DISK.from_pretrained(checkpoint='depth', device=device)
+        inp = torch.ones(1, 3, 256, 256, device=device)
+        output = disk(inp)
+        assert output is not None
+
+    def test_smoke_epipolar(self, device):
+        disk: DISK = DISK.from_pretrained(checkpoint='epipolar', device=device)
+        inp = torch.ones(1, 3, 256, 256, device=device)
+        output = disk(inp)
+        assert output is not None
+
```

Comment:
btw, would be nice if we can add a `dynamo` test here. E.g https://github.com/kornia/kornia/blob/master/test/filters/test_median.py#L82

Reply:
@johnnv1 @jatentaki good point about dynamo, let's do it as a separate PR, as we are time-constrained to get DISK into kornia before IMC2023 starts.